smdebug/tensorflow/keras.py

```diff
@@ -794,9 +832,16 @@ def _save_layer_values(self, logs):
                 # Layer Inputs are flattened and passed as a list into
                 # the next layer. Unpacking it speeds up the _make_numpy fn.
                 layer_input = layer_input[0]
-            layer_input_tensor_name = get_export_name_for_keras(str(layer_name), "input")
+            layer_name = str(layer_name)
+            idx = layer_name_dict.get(layer_name, 0)
```

Comment:
will this be 0 always because this is the first time the dict has been accessed after L820?